src/clr-addons/quick-list/quick-list.component.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { CLR_BLANK_OPTION, ClrQuickListValue } from './add-option/add-option.service';
+
+@Component({
+  selector: 'clr-quick-list',
+  templateUrl: './quick-list.component.html',
+  styleUrls: ['./quick-list.component.scss'],
+})
+export class ClrQuickListComponent<T> {
+  @Input() blankOption: ClrQuickListValue<T> = CLR_BLANK_OPTION;
```

Comment:
all public input and output parameters need to be prefixed with clr according to our styleguide. Only needed in this class, as the option component is not public
e.g. @Input('clrBlankOption') blankOption

Reply:
done